app/views/shared/_search_navbar_services_menu.html.erb

```diff
@@ -1,11 +1,11 @@
 <ul class="dropdown-menu sul-services-menu" role="menu" aria-labelledby="searchNavbarTopMenu" id="libraryServicesMenu">
-  <li><span class="dropdown-item-text text-uppercase text-secondary fw-semibold"><small>Need assistance?</small></span></li>
-  <li><a href="https://library.stanford.edu/contact-us" class="dropdown-item" role="menuitem">Ask a librarian</a></li>
-  <li><a href="https://guides.library.stanford.edu/searchworks" class="dropdown-item" role="menuitem">SearchWorks user guide</a></li>
-  <li><a href="https://guides.library.stanford.edu/c.php?g=1081009&p=7878021" class="dropdown-item" role="menuitem">Access to online articles</a></li>
-  <li><a href="https://library-status.stanford.edu/" class="dropdown-item" role="menuitem">Check system status</a></li>
+  <li role="presentation"><span class="dropdown-item-text text-uppercase text-secondary fw-semibold"><small>Need assistance?</small></span></li>
```

Comment:
If a `role="presentation"` wraps a `role="menuitem"`, does that remove the menuitem from the accessibility tree?

Reply:
Good enough.